src/components/Dialog/SettingDialog/FileNameTemplateDialog.vue

```diff
@@ -78,50 +78,51 @@
 <script setup lang="ts">
 import { computed, ref, nextTick } from "vue";
 import { QInput } from "quasar";
-import { useStore } from "@/store";
-import {
-  buildAudioFileNameFromRawData,
-  DEFAULT_AUDIO_FILE_BASE_NAME_TEMPLATE,
-  replaceTagIdToTagString,
-  sanitizeFileName,
-} from "@/store/utility";
+import { replaceTagIdToTagString, sanitizeFileName } from "@/store/utility";
+import { UnreachableError } from "@/type/utility";
 
 const props = defineProps<{
   openDialog: boolean;
+  defaultTemplate: string;
+  availableTags: (keyof typeof replaceTagIdToTagString)[];
+  currentTemplate: string;
```

Comment:
やっぱりちょっと引数名難しいですね！
なんでこの引数が分かりづらいかというと、currentは相対的な概念であって、誰にとってのいつの「現在」かわかんないなんですよね。
だからどういう「現在の値」を指定すればいいのかコメント書くのが良さそう。

あるいはソフトウェアにとっての絶対的な概念ということで、savedTempalteとかでも良いかもです。
```suggestion
  savedTemplate: string;
```